server/pulp/server/agent/connector.py

```diff
@@ -0,0 +1,29 @@
+from gofer.messaging import Connector
+
+from pulp.server.config import config
+
+
+def get_url():
+    """
+    This constructs a gofer 2.x URL and is intended to maintain
+    configuration file backwards compatibility until pulp 3.0
```

Comment:
I don't think we've committed anything for 3.0 specifically, so I'll suggest a future, major Pulp release.